src/apps/api/views/competitions.py

```diff
@@ -583,15 +583,59 @@ def manually_migrate(self, request, pk):
 
     @action(detail=True, url_name='rerun_submissions')
     def rerun_submissions(self, request, pk):
+
+        # Limit for rerunning submissions
+        RERUN_SUBMISSION_LIMIT = 30
```

Comment:
could we implement this as environment variable ? I think it's better that the platform administrator can modified it.